CONTRIBUTING.md

```diff
@@ -257,17 +165,40 @@ into the source repository.
   area of the PR. You can also advertise the PR on devel@riot-os.org mailing
   list and ask for a review there.
 
+* Try to answer reviews as quickly as possible to speed up the review process
+  and avoid stalled PRs.
+
 You can find more information about RIOT development procedure on this
 [wiki page][development-procedures].
 
+[about-pull-requests]: https://help.github.com/articles/about-pull-requests/
+[development-models]: https://help.github.com/articles/creating-a-pull-request-from-a-fork
 [existing-pull-requests]: https://github.com/RIOT-OS/RIOT/pulls
 [archived-pull-requests]: https://github.com/RIOT-OS/RIOT/pulls?q=is:pr+label:"State:+archived"
-[coding-conventions]: CODING_CONVENTIONS.md
 [uncrustify]: http://uncrustify.sourceforge.net
 [development-procedures]: https://github.com/RIOT-OS/RIOT/wiki/Development-procedures
 
-### Git usage
+## Writing Documentation
+[writing-documentation]: #writing-documentation
+
+Documentation improvements are always welcome and a good starting point for
+new contributors. This kind of contribution is merged quite quickly in general.
+
+RIOT documentation is built with [doxygen][doxygen]. Doxygen is configured to
+parse header (.h) and `doc.txt` files in the RIOT source code to generate
+the modules, cpus, boards and packages documentation.
+General documentation pages are written in Markdown and located in
+`doc/doxygen/src`.
+
+To generate the documentation, simply run `make doc`
```

Comment:
```
$ make doc
```

is consistent with layout in previous sections and makes it easier for the reader to spot the important commands.

Reply:
Ack. Fixed!